README.md

```diff
@@ -15,7 +15,7 @@ Java idiomatic client for [Google Cloud Platform][cloud-platform] services.
 This client supports the following Google Cloud Platform services at a [GA](#versioning) quality level:
 -  [Stackdriver Logging](#stackdriver-logging-ga) (GA)
 -  [Cloud Datastore](#google-cloud-datastore-ga) (GA)
--  [Cloud Storage](#google-cloud-storage-ga) (GA)
+-  [Cloud Storage](https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-storage) (GA)
```

Comment:
This  breaks the "isolation" of the forks. Is it possible to do something like just `#tree/master/google-cloud-storage` ?